### DISCUSSION TITLE
Where can we read about the docs of latest v1.0.0-dev.x version?

### DISCUSSION THREAD

**Discussion by vigneshksaithal** [rrousselGit/riverpod#656] [Q&A] [answered]
Interested in testing out latest version.

**Comment by wujek-srujek** [comment]
You are out of luck for now, the official docs is not up-to-date yet, and the API docs aren't fully written either. Your best bet is to just upgrade to the latest and fix the code by looking at Riverpod code. Not perfect, but this is what I did (there aren't so many changes) and it worked for me.

**Reply by vigneshksaithal** [reply]
Can you tell me what all has changed because, I am beginner to riverpod. For me context.read doesn't work. What is the alternaive for that?

**Reply by davidmartos96** [reply]
Here is all the information: https://github.com/rrousselGit/river_pod/issues/335

**Comment by vigneshksaithal** [comment, marked as answer]
Answer to this question is in this issue - #335 .

Thanks @davidmartos96 🙏.
